src/modules/jcms_rest/tests/src/Functional/ValidQueryEndpointTest.php

```diff
@@ -26,6 +26,15 @@ public function validQueryEndpointProvider() : array {
         400,
         'Invalid containing parameter',
       ],
+      'containing - promotional-collections' => [
+        '/collections?containing[]=article/1&containing[]=blog-article/2&containing[]=digest/3&containing[]=event/4&containing[]=interview/5',
```

Comment:
this query doesn't mention `promotional-collections`, only `collections` and 5 allowed simpler content types?

Reply:
Will fix, the containing parameter will trigger an error if a single filter value is not supported